integrations/groups-taxonomy.php

```diff
@@ -109,25 +128,24 @@ function dt_push_groups() {
 					}
 				}
 
-				$key = array_search( $group, $connection_map, true );
-				if ( ! in_array( $group, $successed_groups, true ) ) {
-					$successed_groups[] = $group;
-					update_post_meta( $post->ID, 'dt_connection_groups_pushed', $successed_groups );
-				}
-				if ( false !== $key || null !== $key ) {
-					unset( $connection_map[ $key ] );
+				if ( ! in_array( $group, $succeeded_groups, true ) ) {
```

Comment:
@armdev17 Can we have only one block, where will update `dt_connection_groups_pushed` ?